src/SFA.DAS.FAT.Web/wwwroot/js/site.js

```diff
@@ -428,27 +441,27 @@ ShowHidePanels.prototype.showPanel = function(panel) {
 }
 
 
-var showHidePanels = document.querySelectorAll('[data-show-hide-panels]');
+let showHidePanels = document.querySelectorAll('[data-show-hide-panels]');
 nodeListForEach(showHidePanels, function (showHidePanel) {
   new ShowHidePanels(showHidePanel).init();
 });
 
-var feedbackGraphs = document.querySelectorAll('[data-feedback-graph]');
+let feedbackGraphs = document.querySelectorAll('[data-feedback-graph]');
 nodeListForEach(feedbackGraphs, function (feedbackGraph) {
   new FeedbackGraph(feedbackGraph).init();
 });
 
 
-var courseProvidersOrderBy = document.getElementById('course-providers-orderby');
+let courseProvidersOrderBy = document.getElementById('course-providers-orderby');
 
 courseProvidersOrderBy.addEventListener('change', function() {
-   document.getElementById("course-providers-form").submit();
+   document.getElementById("course-providers-order-form").submit();
 });
 
 
-var buttonDeliveryModeProvider = document.getElementById('filteritem-modes-filter-Provider')
-var buttonDeliveryModeDayRelease = document.getElementById('filteritem-modes-filter-DayRelease')
-var buttonDeliveryModeBlockRelease = document.getElementById('filteritem-modes-filter-BlockRelease')
+let buttonDeliveryModeProvider = document.getElementById('filteritem-modes-filter-Provider')
+let buttonDeliveryModeDayRelease = document.getElementById('filteritem-modes-filter-DayRelease')
+let buttonDeliveryModeBlockRelease = document.getElementById('filteritem-modes-filter-BlockRelease')

```

Comment:
just a note, ?I have found that using let sometimes means the javascript throws issues, but for no obvious reason.  I did have a number of these already using let but I found them to work inconsistently occasionally, remedied by using var instead.  I'm happy for this to stay as is if there are no issues.

Reply:
I think the newer way is to use let. The compiler was throwing warnings on using var.